cmd/krel/cmd/changelog.go

```diff
@@ -205,22 +205,26 @@ func runChangelog(opts *changelogOptions, rootOpts *rootOptions) error {
 	}
 	defer func() {
 		if err := repo.Checkout(currentBranch); err != nil {
-			logrus.Errorf("unable to restore branch %s: %v", currentBranch, err)
+			logrus.Errorf("restore branch %s: %v", currentBranch, err)
 		}
 	}()
 
+	logrus.Infof("Checking out %s branch", git.Master)
 	if err := repo.Checkout(git.Master); err != nil {
 		return errors.Wrap(err, "checking out master branch")
 	}
 
+	logrus.Info("Writing markdown")
 	if err := writeMarkdown(repo, toc, markdown, tag); err != nil {
 		return err
 	}
 
+	logrus.Info("Writing HTML")
 	if err := writeHTML(opts, tag, markdown); err != nil {
 		return err
 	}
 
+	logrus.Info("Commiting changes")
```

Comment:
```suggestion
	logrus.Info("Committing changes")
```
I think this should fix linting :)

Reply:
Ah thank you for the hint, fixed that :+1: